### PR TITLE
Epmlstrtai 687

### PR DESCRIPTION
**Pull request by Walrus94**
Moved adding default step suggestions from configs and method in service to liquibase changelog, removed redundant method

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 1]
```diff
@@ -10,4 +10,5 @@
     <include file="db/changelog/db.changelog-entities.xml"/>
     <include file="db/changelog/db.changelog-references.xml"/>
     <include file="db/changelog/db.changelog-indexes.xml"/>
+    <include file="db/changelog/db.changelog-defaultData.xml"/>
```
Comment (pomkine) [inline src/main/resources/db/changelog/db.changelog-master.xml]: hm.. not sure if it's a good name for a default step suggestions changelog - _db.changelog-defaultData.xml_ . Maybe we can name it _db.changelog-defaultStepSuggestions.xml_ or something like that?

Comment (hdghg) [inline src/main/resources/db/changelog/db.changelog-master.xml]: This kind of data usually called seed https://en.wikipedia.org/wiki/Database_seeding
Select appropriate name i.e. db.changelog.stepSuggestionSeed.xml

File: src/main/resources/db/changelog/db.changelog-defaultData.xml [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+            <column name="CONTENT" value="&quot;link &quot;&lt;linkName&gt;&quot; from &quot;&lt;containerName&gt;&quot; match  &quot;&lt;regex&gt;&quot;&quot;"/>
+            <column name="TYPE" value="THEN"/>
+        </insert>
+        <insert tableName="STEPSUGGESTION">
+            <column name="CONTENT" value="&quot;button &quot;&lt;linkName&gt;&quot; from &quot;&lt;containerName&gt;&quot; match  &quot;&lt;regex&gt;&quot;&quot;"/>
+            <column name="TYPE" value="THEN"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="" id="default_stepsugesstions_and">
```
Comment (hdghg) [inline src/main/resources/db/changelog/db.changelog-defaultData.xml]: correct spelling is suggestion

Comment (Walrus94) [reply src/main/resources/db/changelog/db.changelog-defaultData.xml]: fixed

File: src/main/resources/db/changelog/db.changelog-master.xml (hunk 1 above)
Comment (Walrus94) [reply src/main/resources/db/changelog/db.changelog-master.xml]: fixed

File: src/main/resources/db/changelog/db.changelog-stepSuggestionSeed.xml [hunk 3]
```diff
@@ -0,0 +1,230 @@
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+   http://www.liquibase.org/xml/ns/dbchangelog
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="" id="default_stepsugestions_given">
```
Comment (hdghg) [inline src/main/resources/db/changelog/db.changelog-stepSuggestionSeed.xml]: correct spelling is suggestion

**Pull request merged**